communities/microgalaxy/lab/sections/3_microbiome.yml

```diff
@@ -1,9 +1,221 @@
 id: microbiome
 title: Microbiome
 tabs:
+- id: overview
+  title: Overview
+  heading_md: >
+    Microbiome data analysis can encompass metabarcoding, metagenomics, metatranscriptomics, and metaproteomics. These analyses (from raw data to differential analyses or visualization via assembly) are supported in microGalaxy with different tool suites and potential analyses available on microGalaxy. 
+  content:
+    - title_md: "Figure: Overview of microbiome data analysis tasks with corresponding tool suites and potential analysis available on microGalaxy"
+      description_md: |
+        <div>
+          <img src="{{ figure_microbiome_url }}" alt="Overview of microbial data analysis" style="max-width: 100%;">
+          <p style="font-size: 0.9em;">
+
+            Outputs are color-coded by type and numbers indicate available tool suites for each step.
+
+            <strong> Input data (blue)</strong> represent raw datasets, including sequencing reads and mass spectra.  
+            <strong> Intermediate outputs (gray)</strong> include cleaned reads, assemblies, and OTUs/AVSs.  
+            <strong> Taxonomic outputs (green)</strong> represent taxonomic profiling results.  
+            <strong> Functional outputs (yellow)</strong> highlight functional profiling and differential feature analysis.  
+            <strong> Other outputs (orange)</strong> indicate annotated genomes, reconstructed genomes, and antibiotic resistance genes.
+
+            <strong> Numbers correspond to tool suites:</strong>  
+            (1) [<code>FastQC</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc), [<code>Cutadapt</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt), [<code>fastp</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp), MultiQC, [<code>Porechop</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/porechop/porechop), [<code>Filtlong</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/filtlong/filtlong), [<code>NanoPlot</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/nanoplot/nanoplot), [<code>SortMeRNA</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/rnateam/sortmerna/bg_sortmerna), [<code>Bowtie</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/devteam/bowtie2/bowtie2), [<code>minimap</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/minimap2/minimap2)  
```

Comment:
The links do not look like links:

<img width="888" alt="Screenshot 2025-02-13 at 09 45 15" src="https://github.com/user-attachments/assets/d8bf309b-97f5-4339-9ec5-303b6cdcfcd6" />

Reply:
that was just a push to save the current status, now this figure is finished